src/server/game/Battlefield/Battlefield.h

```diff
@@ -179,7 +180,7 @@ class TC_GAME_API Battlefield : public ZoneScript
         GuidUnorderedSet _players[PVP_TEAMS_COUNT];
         GuidUnorderedSet _playersInWar[PVP_TEAMS_COUNT];
         GuidUnorderedSet _groups[PVP_TEAMS_COUNT]; // contains the two different raid groups
-        GuidVector _playerQueue[PVP_TEAMS_COUNT];
+        std::deque<ObjectGuid> _playerQueue[PVP_TEAMS_COUNT];
```

Comment:
fyi - https://github.com/TrinityCore/TrinityCore/blob/master/src/server/game/Entities/Object/ObjectGuid.h#L344
this also makes the include unneeded

Reply:
ow, didnt notice, ty